tests/server/jobs/test_jobs.py

```diff
@@ -247,6 +247,19 @@ def transient_err_fun(tmp_dir: str, succeed_on_nth_run: int):
     raise TransientError(RuntimeError("test transient error."))
 
 
+@job(max_workers=1, use_process=False, transient_error_classes=["builtins.TimeoutError"])
```

Comment:
```suggestion
@job(max_workers=1, use_process=False, transient_error_classes=[TimeoutError])
```

is there a reason we can't support this?